libevmcore/Assembly.cpp

```diff
@@ -156,6 +163,130 @@ ostream& Assembly::stream(ostream& _out, string const& _prefix, StringMap const&
 	return _out;
 }
 
+Json::Value Assembly::createJsonValue(string _name, int _locationX, int _locationY, string _value, string _jumpType) const
+{
+	Json::Value value;
+	assert(!_name.empty());
```

Comment:
Please do not use assert, it will "crash" the whole process if it fails. It is probably not too bad if the name is an empty string.